lib/TaxId.php

```diff
@@ -12,7 +12,6 @@
  * @property string $country
  * @property int $created
  * @property string $customer
- * @property bool $deleted
```

Comment:
Technically that one is possible -- all deleted resources are represented as `{"id": "foo_123", "deleted": true}` in the API. That said it doesn't look like we document this attribute on other deletable resources, and there is a documented helper for it anyway (https://github.com/stripe/stripe-php/blob/master/lib/StripeObject.php#L519) so let's be consistent and remove it here as well.